tests/pipeline-all-amps/docker-entrypoint.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+set -e
+
+ALFRESCO_WEBAPP_DIR=$TOMCAT_DIR/webapps/$ALFRESCO_WEBAPP
+ALFRESCO_MMT_JAR=$TOMCAT_DIR/alfresco-mmt/alfresco-mmt*.jar
+
+echo Available AMPs in $ALFRESCO_AMPS_DIR
+ls -l $ALFRESCO_AMPS_DIR
+
+echo Previously installed AMPs in $ALFRESCO_WEBAPP_DIR
+INSTALLED_AMPS=$(java -jar $ALFRESCO_MMT_JAR list $ALFRESCO_WEBAPP_DIR)
+echo "$INSTALLED_AMPS"
+INSTALLED_AMPS=$(echo "$INSTALLED_AMPS" | grep ^Module | cut -f2 -d' ' | tr -d "'")
+
+echo Uninstall previously installed AMPs in $ALFRESCO_AMPS_DIR
+for AMP in $INSTALLED_AMPS
+do
+	echo Uninstalling $AMP
+	java -jar $ALFRESCO_MMT_JAR uninstall $AMP $ALFRESCO_WEBAPP_DIR
```

Comment:
Historically this hasn't been very reliable, unfortunately some AMPs overwrite files and uninstalling can cause issues unless you have a backup, which won't be the case here.

Some of the AMPs pre-installed are also required i.e. share-services and trash can cleaner. So do you really need to do this?

Reply:
I too think it would be far better to start with the simplest Repo image and always ADD to it. Uninstall is really not supported and we will always be asking is it really the same as just adding onto the base.

Reply:
Another approach, I have been thinking about is putting the various amps into a different directory and the providing a way to add selected ones to the war. At least then the image contains everything to start with.

Reply:
I wasn't aware of the base repo image, will use that one and remove the uninstall workaround

Reply:
the base image seem to be built only for latest, so it looks like it can't be used

Reply:
the amp jars are all included in a dir but installed in the webapp only if named in the ALFRESCO_AMPS whose default value is the full list so I guess @alandavis that's already what you would like